mintlify/mint
# Terminate CLI port properly

## Summary
Apparently using `spawn` is better for terminating ports properly than `exec` ([stackoverflow](https://stackoverflow.com/questions/20187184/how-to-kill-childprocess-in-nodejs)).

## Test Plan
1. Run `mintlify dev`
2. Terminate it
3. Run `lsof -i:3000`. There should be no node processes running. I think it's ok if the Google runs are running.
![image](https://user-images.githubusercontent.com/55263191/213018688-4e248514-7e65-4678-b4bc-7383dd6dcf65.png)
